interview-questions.md

```diff
@@ -15,43 +15,58 @@ Researched answer:
 1. What is git? What is the difference between git and Github?
 
 Your answer:
+Global Information Tracker. Git is on your local machine, but Github is on the world-wide-web.
 
 Researched answer:
+Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
 Your answer:
+Operators used in conditional statements. 
 
 Researched answer:
+Equality operators, relational operators, and logical operators.
 
 3. What is an index? What is the difference between index and value?
 
 Your answer:
+An index is an address or position. Values are types of data.
 
 Researched answer:
+The index of a VALUE in an array or string is that VALUE's location within the array or string.
 
 4. What is iteration?
 
 Your answer:
+Another way to say Loop. 
 
 Researched answer:
+Repeating computations.
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
 Your answer:
+A team of two with a driver and a navigator. The driver handles the typing while the navigator instructs as well as colaborates with the driver on what to type.
 
 Researched answer:
+Pair programming is an efficient way for two programmers to work together at one workstation and develop software. One, the driver, writes code while the other (the navigator) reviews, proofreads and checks the code as he or she writes it. The two programmers switch roles whenever needed.
 
 ## Looking Ahead: Terms for Next Week
 
```

Comment:
It's always great to look ahead at the terms so you have some context when hearing the terms during lectures.